xray/manager.go

```diff
@@ -39,40 +39,78 @@ func (sm *XrayServicesManager) Client() *jfroghttpclient.JfrogHttpClient {
 	return sm.client
 }
 
-// GetVersion will return the xray version
+// GetVersion will return the Xray version
 func (sm *XrayServicesManager) GetVersion() (string, error) {
 	versionService := services.NewVersionService(sm.client)
 	versionService.XrayDetails = sm.config.GetServiceDetails()
 	return versionService.GetVersion()
 }
 
-// CreateWatch will create a new xray watch
+// CreateWatch will create a new Xray watch
 func (sm *XrayServicesManager) CreateWatch(params utils.WatchParams) (*http.Response, error) {
```

Comment:
This is relevant for all Xray APIs - 
I don't think that the go API should expose the ```http.Response``` as a return value value of the API. This is "too low level" for such an API. If the API failed to create the watch (in this example), an error shopuld be returned.
Yuo can look at the older APIs (which manage repositories for example), which also do not expose the http response.
Let's modify the README examples accordingly.